src/conf/order.rs

```diff
@@ -254,6 +254,38 @@ where
     }
 }
 
+/// Checks if the value is an IP address and returns a canonical representation of the address,
+/// reallocated on the configuration pool if necessary.
+fn parse_ip_identifier(
```

Comment:
Just a thought. Maybe it should be named as `as_canonical_ip_identifier()` to better reflect what it does.